athlinks-ontology.ttl

```diff
@@ -367,10 +367,30 @@ dc:date rdf:type owl:AnnotationProperty .
             rdfs:label "Swim Action"@en .
 
 
-###  http://schema.org/GenderType
-schema:GenderType dc:creator "nextdude"^^xsd:string ;
-                  dc:date "2017-05-22T06:00:00Z"^^xsd:dateTime .
 
+###  http://adp.athlinks.com/schema/BracketRanking
+:BracketRanking rdf:type owl:Class ;
```

Comment:
@SmedbergM There are existing classes for AthleteResult (aka bracketranking), AthleteResultList (aka leaderboard) and AthleteCohort (aka bracket). Take a look and see if those cover what you need. They still need properties.

Reply:
The reason I didn't use those is that there's supposed to be a many-to-one relationship between entries and AthleteResultList and AthleteCohort. Here, each BracketRanking should have exactly one entry pointing to it.